src/eslint-config-adeira/__tests__/__snapshots__/presets.test.js.snap

```diff
@@ -811,7 +811,6 @@ Object {
     "@next/next/no-page-custom-font": 2,
     "@next/next/no-script-component-in-head": 2,
     "@next/next/no-script-in-document": 2,
-    "@next/next/no-script-in-head": 2,
```

Comment:
Probably related to this: https://github.com/vercel/next.js/commit/61ea8efe426dab4ebe0e849a29a5dc65b15a95e6 (breaking change in a minor version)